packages/framework-provider-azure-infrastructure/src/infrastructure/stacks/schedules-functions.ts

```diff
@@ -0,0 +1,61 @@
+import { BoosterConfig, ScheduleInterface } from '@boostercloud/framework-types'
+import { FunctionDefinition } from '../types/functionDefinition'
+
+interface TimeTriggerFunction {
+  name: string
+  config: {
+    bindings: [
+      {
+        type: string
+        name: string
+        direction: string
+        schedule: string
+      }
+    ]
+    scriptFile: string
+    entryPoint: string
+  }
+}
+
+export class SchedulesFunctions {
+  public constructor(readonly config: BoosterConfig) {}
+
+  public getFunctionDefinitions(): Array<FunctionDefinition> | undefined {
+    if (Object.keys(this.config.scheduledCommandHandlers).length) {
+      return Object.keys(this.config.scheduledCommandHandlers).map((scheduledCommandName) => {
+        const scheduledCommandMetadata = this.config.scheduledCommandHandlers[scheduledCommandName]
+        const cronExpression = SchedulesFunctions.createCronExpression(scheduledCommandMetadata.scheduledOn)
+        return {
+          name: `scheduleFunction-${scheduledCommandName}`,
+          config: {
+            bindings: [
+              {
+                type: 'timerTrigger',
+                name: `${scheduledCommandName}`,
+                direction: 'in',
+                schedule: `${cronExpression}`,
+              },
+            ],
+            scriptFile: '../dist/index.js',
+            entryPoint: this.config.scheduledTaskHandler.split('.')[1],
+          },
+        } as TimeTriggerFunction
+      }) as Array<FunctionDefinition>
+    }
+    return undefined
+  }
+
+  /**
+   * Azure Functions uses the NCronTab library to interpret NCRONTAB expressions.
+   * An NCRONTAB expression is similar to a CRON expression except that it includes an additional sixth field at the beginning to use for time precision in seconds:
+   * {second} {minute} {hour} {day} {month} {day-of-week}
+   * @param scheduledCommandMetadata
+   * @private
+   */
+  private static createCronExpression(scheduledCommandMetadata: ScheduleInterface): string {
+    const { minute = '*', hour = '*', day = '*', month = '*', weekDay = '*' } = scheduledCommandMetadata
+    const expression = `0 ${minute} ${hour} ${day} ${month} ${weekDay}`
+    const neverRunByDefault = '0 0 5 31 2 ? *'
+    return `${expression !== '* * * ? * *' ? expression : neverRunByDefault}`
```

Comment:
I don't know if I fully understand these two last lines. Some comments here:
* `neverRunByDefault` has 7 items, is that correct or the last asterisk should not be there? 
* I get that the idea is to set an impossible date (February 31th) to avoid the function from running, but why is `'* * * ? * *'` invalid? Doesn't that mean "run every second"? Also, I'm not an expert in corn syntax, but I think that this other pattern means the same: ``'* * * * * ?'`.
* Ideally, I'd prefer that invalid cron expressions just avoid creating the resource than creating the functions with an impossible date.

Reply:
You are right, I copy/paste it from the AWS version and didn't check this use case. As you said, if we don't want to run it then is better to avoid creating the resource. Let me fix it

Reply:
About why it could be an invalid configuration I guess it was because it's the default value: 

`const { minute = '*', hour = '*', day = '*', month = '*', weekDay = '*' } = scheduledCommandMetadata`

so we should check if all the fields are `*` but this is valid. I mean, this configuration `0 * * * * *` means to run it each minute which is valid (you can check it here: https://ncrontab.swimburger.net/). 

I would change it to add it always. Thoughts?